greenriver/hmis-frontend
# Make schema updates for VerifiedBy on CLS

## Description

To test:
- Go to a CE project
- Go to an enrollment -> Current Living Situations
- Add a new current living situation. You should see the dropdown for Verified By. It should be a required field and contain all Continuum Projects.
- After submitting, when you view the record, you should see the project you selected.
- Non-CE projects should still be able to collect Current Living Situation and Verified By should not show up in their forms.

[//]: # 'remove if not applicable'
GH issue: https://github.com/open-path/Green-River/issues/6272
Depends on hmis-warehouse PR: https://github.com/greenriver/hmis-warehouse/pull/4587

## Type of change
[//]: # 'remove options that are not relevant'

## Checklist before requesting review
